BlogTemplate.Tests/Pages/RegisterTests.cs

```diff
@@ -0,0 +1,33 @@
+using BlogTemplate.Data;
+using BlogTemplate.Pages.Account;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace BlogTemplate.Tests.Services
+{
+    class SingleUserRegistrationTests
+    {
+        [Fact]
+        public void RegisterUser_CreateSecondUser_RegistrationFails()
+        {
+            // Arrange
+            //SignInManager<ApplicationUser> fakeSignInManager = new SignInManager<ApplicationUser>();
+            //UserManager<ApplicationUser> fakeUserManager = new UserManager<ApplicationUser>();
+            //ILogger<LoginModel> fakeLogger = new ILogger<LoginModel>();
+
+            RegisterModel registerModel = new RegisterModel(null, null, null, null);

```

Comment:
You probably want to provide a UserManager so that you can control how many users you have (i.e. 0 or 1).  In fact, you should have tests to verify behavior both ways.